### ISSUE TITLE
Deprecate using (-1) for fc.Body queries

### ISSUE THREAD

**Issue by MOARdV** [MOARdV/AvionicsSystems#185]
I added this ability as a convenience for IVA configuration, since it allowed the prop maker to do something like `fc.BodyHasOxygen(-1)` to query the body currently being orbited, and also `fc.BodyHasOxygen(3)` or `fc.BodyHasOxygen("Kerbin")`.

Most of those queries are constants - they can never change without restarting the game.  And MAS is capable of treating them as constants, which would mean there is never overhead for querying those values once they're initialized (better for performance).  But ... because the '-1' queries the current body, and that can change as a vessel changes SoI, I can not use the constant or dependent attributes on those methods.

Instead, add a function `fc.CurrentBody()` that will return the current body index.  This change allows me to use 'dependent' attributes on Body methods, so that they will only update if the body index changes.

TL;DR - performance improvement, more important since MAS finally supports menus, so a "target planet" menu becomes very expensive to update.

**Comment by MOARdV** [comment]
I don't need to apply a similar optimization to the nav Waypoint methods.
